src/components/Pickers/PickerItem.Props.ts

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+export interface IPickerItemProps extends React.Props<any> {
+  item: any;
```

Comment:
I don't think it matter. In fact... I would probably suggest to leave it as any or IObjectWithKey.